nix/modules/mangaki.nix

```diff
@@ -258,19 +299,44 @@ in
 
     # User activation script for directory initialization.
     # systemd oneshot for initial migration.
+    systemd.services.mangaki = {
+      after = [ "postgresql.service" ];
+      requires = [ "postgresql.service" ];
+      wantedBy = [ "multi-user.target" ];
+
+      description = "Mangaki service";
+      path = [ pkgs.mangaki.env ];
+      environment.MANGAKI_SETTINGS_PATH = toString configFile;
+      environment.DJANGO_SETTINGS_MODULE = "mangaki.settings";
+
+      serviceConfig = {
+        User = "mangaki";
+        Group = "mangaki";
+
+        StateDirectory = "mangaki";
+        StateDirectoryMode = "0750";
+        WorkingDirectory = "/var/lib/mangaki";
+      };
+
+      # TODO: django-admin runserver bugs out looking like it fails to parse bash
+      script = ''
+        django-admin migrate
+        python ${pkgs.mangaki.src}/mangaki/manage.py runserver
```

Comment:
Not really familiar with django so I'm not sure `django-admin runserver` is failing.

Reply:
I'll take a look to it

Reply:
Okay, I know why: `django-admin runserver` will proceed to a restart of its own executable, but the current exec is a bash file and not a Python file, so it does not parse properly and throws a syntax error.

The fix would be to patch Django's reloader, I guess.

Reply:
We can keep the things as-is and put a comment on the top of it.